data/json/items/comestibles/junkfood.json

```
@@ -1135,6 +1135,20 @@
     "fun": 12,
     "vitamins": [ [ "vitC", 85 ], [ "calcium", 22 ], [ "iron", 15 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "hotdogs_newyork",
+    "name": { "str": "new york wiener" },
+    "copy-from": "hotdogs_cooked",
+    "weight": "568 g",
+    "looks-like": "hotdogs_cooked",
+    "calories": 721,
+    "description": "What's better than a hot dog? A hot dog smothered in ground meat and chopped onions, of course! Tastes better than a normal hot dog.",
```

Needs two spaces between sentences to pass CI check.
```suggestion
    "description": "What's better than a hot dog?  A hot dog smothered in ground meat and chopped onions, of course!  Tastes better than a normal hot dog.",
```